<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AltruisticDonorDialog</class>
 <widget class="QDialog" name="AltruisticDonorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>436</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>346</width>
    <height>436</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>346</width>
    <height>436</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add New AD</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>81</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="donorGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>192</width>
     <height>139</height>
    </rect>
   </property>
   <property name="title">
    <string>Donor Information</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="donorGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="donorNameLabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="donorNameLineEdit">
       <property name="text">
        <string>Donor Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="donorAgeLabel">
       <property name="text">
        <string>Age</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="donorBTLabel">
       <property name="text">
        <string>Blood Type</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="donorAgeSpinBox">
       <property name="maximum">
        <number>130</number>
       </property>
       <property name="value">
        <number>40</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="donorAgeSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="2">
      <spacer name="donorBTSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="donorBTComboBox">
       <item>
        <property name="text">
         <string>O</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AB</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTabWidget" name="morePairInfo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>321</width>
     <height>261</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="donorCharacteristicsTab">
    <attribute name="title">
     <string>Donor Characteristics</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>124</width>
       <height>74</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="donorCharacteristicsGridLayout">
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="donorWeightSpinBox">
        <property name="maximum">
         <double>400.000000000000000</double>
        </property>
        <property name="value">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="donorGenderComboBox">
        <item>
         <property name="text">
          <string>Male</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Female</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="donorHeightLabel">
        <property name="text">
         <string>Height (m)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="donorWeightLabel">
        <property name="text">
         <string>Weight (kg)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="donorGenderLabel">
        <property name="text">
         <string>Gender</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="donorHeightSpinBox">
        <property name="maximum">
         <double>3.000000000000000</double>
        </property>
        <property name="value">
         <double>1.500000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="hlaInfoTab">
    <attribute name="title">
     <string>HLA Information</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>191</width>
       <height>152</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="donorHLAGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="donorALabel">
        <property name="text">
         <string>Donor HLA A</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="donorA1LineEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="donorA2LineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="donorBLineEdit">
        <property name="text">
         <string>Donor HLA B</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="donorB1lineEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="donorB2LineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="donorCWLineEdit">
        <property name="text">
         <string>Donor HLA CW</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="donorCW1LineEdit"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="donorCW2LineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="donorDPLabel">
        <property name="text">
         <string>Donor HLA DP</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="donorDP1LineEdit"/>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="donorDP2LineEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="donorDQLabel">
        <property name="text">
         <string>Donor HLA DQ</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="donorDQ1LineEdit"/>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="donorDQ2LineEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="donorDRLabel">
        <property name="text">
         <string>Donor HLA DR</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="donorDR1LineEdit"/>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="donorDR2LineEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>277</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="donorHLAAdditionalLabel">
        <property name="text">
         <string>Additional Donor Antigens</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="donorHLALineEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>91</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="donorHLADRLayout">
        <property name="title">
         <string>Donor HLA DR</string>
        </property>
        <widget class="QWidget" name="layoutWidget_5">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>51</width>
           <height>65</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="donorHLADRVerticalLayout">
          <item>
           <widget class="QCheckBox" name="donorDR51CheckBox">
            <property name="text">
             <string>DR51</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="donorDR52CheckBox">
            <property name="text">
             <string>DR52</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="donorDR53CheckBox">
            <property name="text">
             <string>DR53</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="donorHLABWLayout">
        <property name="title">
         <string>Donor HLA BW</string>
        </property>
        <widget class="QWidget" name="layoutWidget_6">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>47</width>
           <height>42</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="donorHLABWVerticalLayout">
          <item>
           <widget class="QCheckBox" name="donorBW4CheckBox">
            <property name="text">
             <string>BW4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="donorBW6CheckBox">
            <property name="text">
             <string>BW6</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>layoutWidget_7</zorder>
    <zorder>layoutWidget_9</zorder>
   </widget>
   <widget class="QWidget" name="commentTab">
    <attribute name="title">
     <string>Comments</string>
    </attribute>
    <widget class="QTextEdit" name="commentTextEdit">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>10</y>
       <width>291</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AltruisticDonorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AltruisticDonorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
